src/CommonAssemblyInfo.cs

```diff
@@ -1,3 +1,5 @@
+using System;
 using System.Runtime.InteropServices;
 
+[assembly: CLSCompliant(true)]

```

Comment:
Is this actually true? I mean, there are a number of [rules](https://docs.microsoft.com/en-us/dotnet/standard/language-independence-and-language-independent-components#cls-compliance-rules) to follow in order to be CLS compliant. Since we never worried about it before, I'm not sure we actually followed them...

Reply:
Oh, we already had this on the FakeItEasy assembly. Never mind.